src/main/java/com/google/cloud/tools/app/impl/cloudsdk/internal/sdk/PathResolver.java

```diff
@@ -30,7 +29,7 @@
    *
    * @return Path to Google Cloud SDK or null
    */
-  public Path getCloudSdkPath() throws FileNotFoundException {
+  public Path getCloudSdkPath() {
```

Comment:
Should we expose this as a File return given that it's public API? We can also just do an overhaul of this whole class in another PR.